extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -98,11 +103,17 @@ public function onEditPageShowEditFormFields( \EditPage $editPage, \OutputPage $
 		$templateType = $this->getTemplateTypeForEdit( $editPage->getTitle(), $wgCityId );
 
 		$editPage->addHiddenField([
-			'name' => 'templateClassificationType',
+			'name' => 'templateClassificationTypeCurrent',
 			'value' => $templateType,
 			'type' => 'hidden',
 		]);
 
+		$editPage->addHiddenField([
```

Comment:
This usage causes Fatal Error on Monobook. Please use approach from this fix https://github.com/Wikia/app/pull/8976/files#diff-e9367e7e28b3950efc1f77b31ae7bc06R99